client/blocks/eligibility-warnings/style.scss

```diff
@@ -18,10 +17,53 @@
 		align-items: center;
 		display: flex;
 		gap: 5px;
+		color: var(--color-text-subtle);
+		font-size: 1rem;
 	}
 
 	.card {
 		box-shadow: none;
+
+		.notice {
+			margin-bottom: 0;
+		}
+	}
+
+	.eligibility-warnings__hold-list {
```

Comment:
I moved this CSS from `plugin-details-CTA` so we can centralize everything from `eligibility-warnings` avoiding duplicating code and simplifying the maintenance